app/package.json

```diff
@@ -7,6 +7,11 @@
     "build": "next build",
     "start": "next start",
     "lint": "next lint",
+    "lint:fix": "next lint --fix",
+    "format:check": "prettier --check .",
+    "format": "prettier --write . --list-different",
+    "check": "npm run lint && npm run format:check",
+    "fix": "npm run lint:fix && npm run format",
```

Comment:
Added these scripts I have found really helpful, basic use is you can just run `npm run check` and then `fix` if needed before committing and then we can assure consistent formatting/linting

Reply:
@adamcohenhillel